.github/workflows/ci_docs.yml

```diff
@@ -117,6 +117,8 @@ jobs:
             sphinx-design
             sphinx-gallery
             sphinx_rtd_theme<3.0
+            imagemagick
+            tectonic
```

Comment:
The badges in the documentation are SVG files, and LaTeX doesn't support SVG with errors like:

```
error: pygmt.tex:52276: LaTeX Error: Cannot determine size of graphic in /home/runner/work/pygmt/pygmt/doc/_build/doctrees/images/455a48e4d8f250af2dc5ad447126b4eeac754d52/zenodo.14535921.svg (no BoundingBox).
```
The issue can be fixed by enabling the [`sphinx.ext.imgconverter`](https://www.sphinx-doc.org/en/master/usage/extensions/imgconverter.html) extension, which requires ImageMagick.

As explained in https://www.sphinx-doc.org/en/master/usage/extensions/imgconverter.html, we can also use 

- sphinxcontrib-svg2pdfconverter + librsvg/CairoSVG